ext/rinku/autolink.c

```diff
@@ -293,3 +293,19 @@ sd_autolink__url(
 
 	return link_end;
 }
+
+/* isspace will consider different sets of characters as whitespace depending on
+*  C locale. Usually this is fine, but the main problem character is a
+*  non-breaking space (Unicode: U+00A0). This abstracted helper method checks
+*  both isspace to be true OR for the data char to be a NBS (typed on an OSX
+*  US keyboard with option-space.)
+*/
+int
+isspace_helper(
+	char data)
```

Comment:
Keep short method signatures on one line.  Usually I make them multi-line when it hits 80 columns (but I'm not sure what style we do at GitHub).